### ISSUE TITLE
[Feature Request] Add ’kubel-process-buffer

### ISSUE THREAD

**Issue by RCmerci** [abrochard/kubel#33]
Like ‘magit-process-buffer ($), display all executed kubectl commands.

**Comment by abrochard** [comment]
I just pushed a naive implementation on the `kubel-process-buffer` branch.

https://github.com/abrochard/kubel/tree/kubel-process-buffer

Feel free to check it out. It logs most of the commands, but not the ones executed outside of `kubel--exec`. Also I don't think I will have it log the result of the commands as it can be very long.

**Comment by RCmerci** [comment]
Thanks for your implementation ! 
A few suggestions :
1. maybe 'kubel-process' -> '*kubel-process*' ?
2. add exit-code to logs
3. log the result if exit-code != 0
4. add interactive command 'kubel-process-buffer' and bind to '$' (like magit)

**Comment by abrochard** [comment]
Thank you for the suggestions.
I just pushed most of these changes on the branch. There are a few points to think about more:
- what format to log these calls in? magit has its own mode and stuff but that seems overkill. I did something a bit custom that supports the async nature of some of the calls
- a lot of kubectl calls are wrapped into a `shell-command-to-string` call, which makes it hard to capture the exit code. maybe we can live with it, i'm not sure
